File: payment-page-react-native-functional-integration-demo/screens/Success.js
import React from 'react'
import { View, Text } from 'react-native'

export default function Success() {
    return (
        <View style={{display: "flex", height:"100%", alignItems: "center", justifyContent: "center"}}>
            <Text style={{color: "green", fontSize:30}}>Success :D</Text>
        </View>
    )
}

File: lib/screens/success.dart
import 'package:flutter/material.dart';

class SuccessScreen extends StatelessWidget {
  const SuccessScreen({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Scaffold(
        appBar: AppBar(
          backgroundColor: Theme.of(context).colorScheme.inversePrimary,
        ),
        body: Column(
          children: [
            Expanded(
              flex: 1,
              child: Container(
                color: Colors.blue,
                width: double.infinity,
                alignment: Alignment.centerLeft,
                padding: const EdgeInsets.symmetric(horizontal: 10),
              ),
            ),
            const Expanded(
                flex: 8,
                child: Center(
                  child: Text(
                    "Payment Successful!",
                    style: TextStyle(fontSize: 16, fontWeight: FontWeight.w600),
                  ),
                ))
          ],
        ));
  }
}
